webui/sg_webui_start.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Check if directory exists
+if [ ! -d "sg-view/node_modules" ]; then
+    echo "Kill servers"
+    killall node -q
+    killall sg-history -q
+fi
+echo "VIEW PORT: $2"
```

Comment:
If the lifetime of the container and the lifetime of the web services were the same, then this type of thing is no longer needed (nor is `sg_webui_stop.sh` below... it can just be a `docker kill`); I guess there would still be a need to check if one of the web services crashes and then exit so that the Docker container dies when one of the web services dies...